sdk/eventgrid/azure-eventgrid/samples/consume_samples/e2e_samples/consume_cloud_events_from_service_bus_queue.py

```diff
@@ -0,0 +1,37 @@
+import sys
+import os
+
+PACKAGE_PARENT = '..'
+SCRIPT_DIR = os.path.dirname(os.path.realpath(os.path.join(os.getcwd(), os.path.expanduser(__file__))))
```

Comment:
So I think I know why this is here, and _it's kinda a neat idea_ since running these samples ad-hoc (given the fact that they use real-package-paths from a subdir) often requires making sure you're in a proper env/path to actually resolve these imports can sometimes be a stumbling block.  however, I worry that if we start linking these samples to users, they'll be like "why would I have that."

Assuming these samples can be run as "self-contained-one-off-scripts" within a properly configured environment without this path, I'd probably yank this; although I feel bad saying this since it is a nice dev convenience layer.  Let me know if you have any clever ways to have our cake and eat it too, or if I'm missing something obvious.